### PR TITLE
Woosung

### PR DESCRIPTION
**Pull request by gomgomigom**
gpu 사용 가능하다면 임베딩할때 gpu사용하게 변경 / gpu 사용 불가시 cpu로 처리함 

모든 과정을 코랩에서 돌려볼 수 있는 shootingstar.ipynb 추가 

accuracy 평가를 할 수 있는 eval.py 추가

### CODE REVIEW

File: module/llmvdb/customdataset.py [hunk 1]
```diff
@@ -1,27 +1,42 @@
 import json
 from tqdm.auto import tqdm
+from torch.utils.data import Dataset
```
Comment (hyogg) [inline module/llmvdb/customdataset.py:3]: Test Comment

**Review by hyogg — APPROVED** [review]
LGTM
